source/modules/user-settings/authorized-substitutes.rst

```diff
@@ -0,0 +1,8 @@
+Authorized substitutes
+======================
+
+Substitutes are user accounts that are authorized to act on behalf of an other user to validate or refuse a ticket or a change.
+
+The user can choose one or several accounts to delegate the validation task. He may optionnaly limit the date range of the delegation by setting one or both date lmimits.
+
+The user may be also a delegatee of an other user. In this case, he will be see in a table his delegators and the date range of each delegation.
```

Comment:
```suggestion
The user may also be a delegatee of another user. In this case, they will see a table of their delegators and the date range of each delegation.
```

Reply:
```suggestion
The user may also be a delegatee of another user. In this case, he will see  a table of his delegators and the date range of each delegation." 
```

Again, i don't think that "he should be replaced with "they" for the same reason. If there is an ambiguity, then i should find an other way to describe the features.